src/dal/Jalasoft.TeamUp.Resumes.DAL/SkillsApiRepository.cs

```diff
@@ -0,0 +1,69 @@
+﻿namespace Jalasoft.TeamUp.Resumes.DAL
+{
+    using System.Collections.Generic;
+    using Jalasoft.TeamUp.Resumes.DAL.Interfaces;
+    using Jalasoft.TeamUp.Resumes.Models;
+    using Newtonsoft.Json;
+    using RestSharp;
+
+    public class SkillsApiRepository : ISkillsRepository
+    {
+        public IEnumerable<Skill> GetSkills(string name)
+        {
+            string emsiSkills = this.SkillsManager(name);
+            var response = JsonConvert.DeserializeObject<Root>(emsiSkills);
+            List<Skill> listSkills = new List<Skill>();
+            if (listSkills != null)
```

Comment:
in line 15 you are instantiating a new listSkills and in line 16 you are asking if that is not null, it will always be different to null, is that intended? or am I missing something?